### DISCUSSION TITLE
Welcome to AirTrail Discussions!

### DISCUSSION THREAD

**Discussion by johanohly** [johanohly/AirTrail#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

🔗 If you want to report a bug, please [open an issue](https://github.com/johanohly/AirTrail/issues/new/choose) instead.

**Comment by tlai88** [comment]
Suggestions: 

- Add Localisation of Date format
DD.MM.YY
DD.MM.YYYY
YYYY.MM.DD

- Trip grouping 
-
